hibernate-core/src/test/java/org/hibernate/orm/test/cascade/RefreshTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.orm.test.cascade;
+
+import java.sql.PreparedStatement;
+import java.util.Date;
+import java.util.Iterator;
+
+import org.hibernate.engine.spi.SessionImplementor;
+
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.FailureExpected;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+/**
+ * Implementation of RefreshTest.
+ *
+ * @author Steve Ebersole
+ */
+@DomainModel(
+		xmlMappings = {
+				"org/hibernate/orm/test/cascade/Job.hbm.xml",
+				"org/hibernate/orm/test/cascade/JobBatch.hbm.xml"
+		}
+)
+@SessionFactory
+@FailureExpected( reason = "This should be fixed by Nathan PR")
```

Comment:
This is the PR https://github.com/hibernate/hibernate-orm/pull/3250 fixing this test

Reply:
thanks for mentioning me. I am moved by my name showing up here, :).